src/vs/base/common/event.ts

```diff
@@ -148,6 +148,7 @@ export namespace Event {
 
 					if (leading && !handle) {
 						emitter.fire(output);
+						output = undefined;
```

Comment:
I'm guessing this will break the current usage of `leading`: https://github.com/microsoft/vscode/commit/f1abc274df165aea3fd83720f7ff6a9c4bb8c8cd#r22787803

Leaving the review for @roblourens

Reply:
I think that the idea is just to clear the output after each time it is emitted? Otherwise the first output will be received twice if leading=true. I think that's correct.

Reply:
@roblourens @alexr00 So it should work, and there are no changes required, right? Can I resolve it then?

Reply:
Yes

Reply:
Thanks!